terraform/environment/region/modules/app/ecs.tf

```diff
@@ -236,17 +236,21 @@ locals {
           value = "https://oidc.integration.account.gov.uk"
         },
         {
-          # this is not the final value, but will allow signin to be tested while the real redirectURL is changed
           name  = "APP_PUBLIC_URL",
-          value = var.app_env_vars.app_public_url
+          value = var.app_env_vars.app_public_url == "" ? local.dev_app_fqdn : var.app_env_vars.app_public_url
+        },
+        {
+          # this is not the final value, but will allow signin to be tested while the real redirectURL is changed
+          name  = "AUTH_REDIRECT_BASE_URL",
```

Comment:
what is this doing?

Reply:
This was added as I need a reference to the application base URL to build a return URL for pay to redirect the user back to. As this was set to the prototype for dev envs it means the pay flow breaks after a user completes their payment info. 

Aaaand I just noticed I didn't push the changes to main that actually use this value 🤦